src/NuProj.ProjectSystem.12/Templates/Projects/NuProj/NuProj.nuproj

```diff
@@ -32,4 +32,10 @@
     <Content Include="Readme.txt" />
   </ItemGroup>
   <Import Project="$(NuProjPath)\NuProj.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+  <Target Name="BeforeBuild">
+  </Target>  
+  <Target Name="AfterBuild">
+  </Target>
+  -->
```

Comment:
I guess the default project template has it to make it clear that you can use it.
If you think it is unneeded I can remove the comment.